src/sentry/api/serializers/models/group.py

```diff
@@ -205,11 +226,11 @@ def get_attrs(
             seen_groups = {}
             subscriptions = defaultdict(lambda: (False, False, None))
 
-        assignees = {
+        assignees: Mapping[int, ActorTuple] = {
```

Comment:
group_id -> actor tuple, not actors.  The typing on this was wrong originally (see `resolve_dict`)